src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -181,7 +182,8 @@ private void setFilingApiData(
       ApiLogger.debug("Value of 'isNoChange' flag for Remove is :" + isNoChange, logMap);
       var updateSubmission = new UpdateSubmission();
       collectUpdateSubmissionData(updateSubmission, submissionDto, passThroughTokenHeader, isNoChange, logMap);
-      setRemoveSubmissionData(userSubmission, updateSubmission, isNoChange, submissionDto.getRemove().getIsNotProprietorOfLand(), logMap);
+      setRemoveSubmissionData(userSubmission, updateSubmission, isNoChange, submissionDto.getEntity().getEmail(),
```

Comment:
just wondering if the submissionDto.getEntity().getEmail() would be present for a no change journey? I think it might be but wasn't sure

Reply:
I've recently filed no change remove submissions (looking at another issue) and the email address is present in the json sent to the Chips-Filing-Consumer - so should be ok.

Reply:
Actually this raises the question, If we're already sending it to the chips-filing-consumer, do we need to include it twice. Could the change be made in the `chips-filing-consumer` to map it from where it currently is: `userSubmission.entity.email`?

Reply:
does it not come through though if on a change journey but change something else, ie. it's not in the 'change' section/block

Reply:
Unresolving. Need to investigate @dlloyd1-ch 's question above and whether the OE API change is actually required.

Reply:
No, API change not required - we can handle this just in chips-filing-consumer as it's now clear that e-mail address is sent through already, on both a change and no change journey.